pkg/kong/kong.go

```diff
@@ -72,39 +69,6 @@ func SimpleProxySetup(kc *kubernetes.Clientset, namespace string) error {
 	return nil
 }
 
-// DeployControllers deploys the Kong Kubernetes Ingress Controller (KIC) and other relevant
-// Ingress controllers to the provided cluster given a *kubernetes.Clientset for it.
-// FIXME: this is a total hack for now
-func DeployControllers(kc *kubernetes.Clientset, containerImage, namespace string) (context.CancelFunc, error) {
```

Comment:
I moved this back into KIC `railgun` because this is a terrible hack and it really doesn't belong here as it's highly specific to testing the KIC.